spring/src/main/java/org/axonframework/spring/authorization/AuthorizationMessageDispatchInterceptor.java

```diff
@@ -0,0 +1,41 @@
+package org.axonframework.spring.authorization;
```

Comment:
I'm afraid we need a copyright statement here. The template is in the project [here](https://github.com/AxonFramework/AxonFramework/blob/master/copyright-template.xml).